src/locales/fi.json

```diff
@@ -274,7 +295,8 @@
       "waste": "Roskat",
       "accessible": "Esteetön paikka",
       "previousLeases": "Edelliset vuokralaiset",
-      "comment": "Huomiot"
+      "comment": "Huomiot",
+      "edit": "Muokkaa"
```

Comment:
I think it has already been added to `common` terms:
```
"common": {
      "edit": "Muokkaa",
```

Reply:
Done.